spec/_attachments/ic.did

```diff
@@ -224,6 +231,21 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // Canister backup and restore
```

Comment:
super nitpick: we're not using the term "backup" and "restore" in the API. I'd try to not introduce new vocabulary in the documentation to prevent confusion.

```suggestion
  // Canister snapshots
```